api/php/DBgateways/RestaurantDbGateway.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+include_once "php/config.php";
+
+class RestaurantDbGateway {
+
+    // private $servername = "localhost";
+    // private $username = "root";
+    // private $password = "foody";
+    // private $db = "foody";
+
+
+    private $dbConnection = null;
+
+    public function __construct() {
+        $this->dbConnection = new DbConnecter();
+    }
+
+    // public function getRestaurants () {
+
+    // }
+
+    public function validateRestaurant (string $restaurantName, string $restaurantAddress, string $restaurantPhone, string $restaurantWebsite) {
+        $restaurantName = $this->dbConnection->cleanInput($restaurantName);
+        $restaurantAddress = $this->dbConnection->cleanInput($restaurantAddress);
+        $restaurantPhone = $this->dbConnection->cleanInput($restaurantPhone);
+        $restaurantWebsite = $this->dbConnection->cleanInput($restaurantWebsite);
+        $restaurantName = $this->dbConnection->cleanInput($restaurantName);
+        $restaurantAddress = $this->dbConnection->cleanInput($restaurantAddress);
+
+        $sql = "CALL foody.sp_validateRestaurant ('$restaurantName', '$restaurantAddress', '$restaurantPhone', '$restaurantWebsite');";
```

Comment:
Line 31: Please put SQL query here instead of using "CALL ...". This makes it difficult for other team members to find what this query is doing because we need to navigate to the dbchanges folder and find the sql file for the specific table. Also, it makes the SQL file too long and difficult to navigate.